src/components/navbars/DashboardNavStudent.jsx

```diff
@@ -0,0 +1,38 @@
+import Home from '../../assets/icons/home.png';
+import Lesson from '../../assets/icons/lessons.png';
+import Messages from '../../assets/icons/messages.png';
+import { NavLink } from 'react-router-dom';
+import Notification from '../../assets/icons/notification.png';
+import Payment from '../../assets/icons/payment.png';
+
+const DashboardNavStudent = () => {
+  const navbarItemsStudent = [
+    { src: Home, text: 'Home', link: '' },
+    { src: Lesson, text: 'My lessons', link: 'lessons' },
+    { src: Notification, text: 'Notifications', link: 'notifications' },
+    { src: Messages, text: 'Messages', link: 'messages' },
+    { src: Payment, text: 'Payments', link: 'payments' },
+  ];
+
+  return (
+    <>
```

Comment:
You can provide more semantic meaning by wrapping it in a `nav` element instead of a fragment. You can also provide additional accessibility support with `aira-label` = `student dashboard`

Reply:
I'll change the tag div to nav in the file DashboardNav.jsx that renders the contents of these components (navbars for student and teacher in the dashboard). Thanks for noticing, forgot about the nav tag.

Reply:
added this to the DashboardNav.jsx

Reply:
added the aria-label (student or teacher dashboard) to the div in the Dashboard.jsx